src/Query.tsx

```diff
@@ -90,22 +88,22 @@ export interface QueryProps<TData = any, TVariables = OperationVariables> {
   query: DocumentNode;
   variables?: TVariables;
   ssr?: boolean;
-}
-
-export interface QueryState<TData = any> {
-  result: ApolloCurrentResult<TData>;
+  displayName?: string;
+  skip?: boolean;
+  client?: ApolloClient<Object>;
```

Comment:
Need to add tests for these props in a follow-up PR in `Query.test.tsx`.

Reply:
@excitement-engineer this kind of stuff does warrant an overall discussion about how we want the test suite to be adjusted / grow with both HOCs and components